packages/core/src/connectivity/scp-cf/authorization-header.spec.ts

```diff
@@ -59,28 +81,21 @@ describe('buildAuthorizationHeaders', () => {
   });
 
   describe('principal propagation', () => {
-    it('does not throw on Principal Propagation', async () => {
-      const destination = {
-        url: '',
-        authentication: 'PrincipalPropagation',
-        proxyType: 'OnPremise',
-        proxyConfiguration: {
-          headers: {
-            'SAP-Connectivity-Authentication': 'someValue',
-            'Proxy-Authorization': 'someProxyValue'
-          }
-        }
-      } as Destination;
-
-      const headers = await buildAuthorizationHeaders(destination);
+    it('builds headers', async () => {
+      const headers = await buildAuthorizationHeaders(
+        principalPropagationDestination
+      );
       checkHeaders(headers);
+    });
 
-      delete destination!.proxyConfiguration!.headers![
-        'SAP-Connectivity-Authentication'
-      ];
+    it('does not throw on Principal Propagation', async () => {
```

Comment:
it says does not throw but in the test it is expected to throw -> confused.

Reply:
Fixed.